### PR TITLE
add support for C* 3.11.10 and make it the default version

### LINKED ISSUE #308: Add support for Cassandra 3.11.10
**Is your feature request related to a problem? Please describe.**
Cassandra 3.11.10 was released a couple days ago. We need to add support for it in k8ssandra.

**Describe the solution you'd like**
Add support for Cassandra 3.11.10 and make it the default version.

**Additional context**
We need https://github.com/datastax/management-api-for-apache-cassandra/issues/66 done first.

### PR DESCRIPTION
**Pull request by jsanda**
**What this PR does**:
Adds support for Cassandra 3.11.10

**Which issue(s) this PR fixes**:
Fixes #308

### CODE REVIEW

File: charts/k8ssandra/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.42.1
+version: 0.43.0
```
Comment (jeffbanks) [inline charts/k8ssandra/Chart.yaml:18]: reminder to bump after update with main.

File: charts/k8ssandra/templates/cassandra/cassdc.yaml [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {{- $datacenter := (index .Values.cassandra.datacenters 0) -}}
 {{- $medusaImage := (printf "%s:%s" .Values.backupRestore.medusa.image.repository .Values.backupRestore.medusa.image.tag) -}}
+
```
Comment (jeffbanks) [inline charts/k8ssandra/templates/cassandra/cassdc.yaml:3]: Nice ...

File: charts/k8ssandra/values.yaml [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+
+  # image Overrides the default image mappings. This is intended for advanced use cases
+  # like development or testing. By default the Cassandra version has to be one that is in
+  # versionImageMap. Template rendering will fail if the version is not in the map. When
+  # you set the image directly, the version mapping check is skipped. Note that you are
+  # still constrained to the versions supported by cass-operator.
+  #
+  # image:
+
   clusterName: k8ssandra
```
Comment (jeffbanks) [inline charts/k8ssandra/values.yaml:29]: looks stale value, should be ""

**Review by jeffbanks — APPROVED** [review]
LGTM, just a couple of comments related to updates to main.  A much cleaner approach to version checks.

File: charts/k8ssandra/values.yaml (hunk 3 above)
Comment (jsanda) [reply charts/k8ssandra/values.yaml:29]: Thanks, looks like I need to rebase.

**Pull request merged**